app/controllers/concerns/hydra/collections_controller_behavior.rb

```diff
@@ -142,6 +142,7 @@ def collection_params
 
     # Queries Solr for members of the collection.
     # Populates @response and @member_docs similar to Blacklight Catalog#index populating @response and @documents
+    # TODO: this should be moved into Hydra::Collections::SearchBehaviors
```

Comment:
This method is calling the search builder: https://github.com/projecthydra-labs/hydra-collections/blob/search-behaviors/app/controllers/concerns/hydra/collections_controller_behavior.rb#L150 So I don't know how it can be move into the class it is calling.